integration-tests/search.test.ts

```diff
@@ -68,6 +68,43 @@ describe('search', () => {
         }
     });
 
+    test('search for various valid parameters in query and request body', async () => {
+        const testPatient: ReturnType<typeof randomPatient> = (await client.post('Patient', randomPatient())).data;
+
+        // wait for the patient to be asynchronously written to ES
+        await waitForResourceToBeSearchable(client, testPatient);
+
+        const aFewMinutesAgo = aFewMinutesAgoAsDate();
+
+        const p = (params: any, postQueryParams: any) => ({
+            url: 'Patient',
+            params: { _lastUpdated: `ge${aFewMinutesAgo}`, ...params },
+            postQueryParams,
+        });
+        const testsParams = [
+            p({ 'address-city': testPatient.address[0].city }, { 'address-country': testPatient.address[0].country }),
+            p(
+                { 'address-postalcode': testPatient.address[0].postalCode },
+                { family: testPatient.name[0].family, 'address-postalcode': testPatient.address[0].postalCode },
+            ),
+            p(
+                { name: testPatient.name[0].given[0] },
+                { name: testPatient.name[0].given[0], gender: testPatient.gender },
```

Comment:
How does the combination work is it an AND or an OR? i.e. if we have name = rob then name = bob what would the result be an OR or an AND?

Reply:
It means AND. If we have name = rob and name = bob it would result in name=[rob, bob] and pass to search, it'll be up for search component to decide what it means.